### DISCUSSION TITLE
Differences compared to Pyrebase

### DISCUSSION THREAD

**Discussion by HenrikPetersen** [AsifArmanRahman/firebase-rest-api#10] [Q&A] [answered]
Hey! Nice to see that someone is continuing the Pyrebase code!

I was wondering if there are any differences or updates compared to pyrebase?
As an example starting a stream_handler for a user with many flags can be a bit slow with pyrebase, did you manage to improve this?

Thank you,
HP

**Comment by AsifArmanRahman** [comment, marked as answer]
Hey, @HenrikPetersen. There are updates, but it's not for the stream_handler part. Updates include implementation of firestore, social logins and some bug fixes for auth and storage. Hopefully in future the mentioned part will be improved when I get the complete grasp of its mechanism.
